docs/en/ingest-management/release-notes/release-notes-8.13.asciidoc

```diff
@@ -0,0 +1,279 @@
+// Use these for links to issue and pulls.
+:kibana-issue: https://github.com/elastic/kibana/issues/
+:kibana-pull: https://github.com/elastic/kibana/pull/
+:beats-issue: https://github.com/elastic/beats/issues/
+:beats-pull: https://github.com/elastic/beats/pull/
+:agent-libs-pull: https://github.com/elastic/elastic-agent-libs/pull/
+:agent-issue: https://github.com/elastic/elastic-agent/issues/
+:agent-pull: https://github.com/elastic/elastic-agent/pull/
+:fleet-server-issue: https://github.com/elastic/fleet-server/issues/
+:fleet-server-pull: https://github.com/elastic/fleet-server/pull/
+
+[[release-notes]]
+= Release notes
+
+This section summarizes the changes in each release.
+
+* <<release-notes-8.13.0>>
+
+Also see:
+
+* {kibana-ref}/release-notes.html[{kib} release notes]
+* {beats-ref}/release-notes.html[{beats} release notes]
+
+// begin 8.13.0 relnotes
+
+[[release-notes-8.13.0]]
+== {fleet} and {agent} 8.13.0
+
+Review important information about {fleet-server} and {agent} for the 8.13.0 release.
+
+[discrete]
+[[security-updates-8.13.0]]
+=== Security updates
+
+{agent}::
+* Update Go version to 1.21.8. {agent-pull}4221[#4221]
+
+[discrete]
+[[breaking-changes-8.13.0]]
+=== Breaking changes
+
+Breaking changes can prevent your application from optimal operation and
+performance. Before you upgrade, review the breaking changes, then mitigate the
+impact to your application.
+
+// copied from Kibana release notes: https://github.com/elastic/kibana/pull/179216
+[discrete]
+[[breaking-176879]]
+.Removes conditional topics for Kafka outputs
+[%collapsible]
+====
+*Details* +
+The Kafka output no longer supports conditional topics. For more information, refer to ({kibana-pull}176879[#176879]).
```

Comment:
```suggestion
*Details* +
The Kafka output no longer supports conditional topics while the final syntax is evaluated ahead of Kafka output GA. For more information, refer to ({kibana-pull}176879[#176879]).
```

I think we want to provide more context on this one, since there is no context in the linked issue.